Sources/LanguageClient/InitializingServer.swift

```diff
@@ -158,10 +184,10 @@ extension InitializingServer: StatefulServer {
 
 extension InitializingServer {
 	/// Run the initialization sequence with the server, if it has not already happened.
-	public func initializeIfNeeded() async throws -> ServerCapabilities {
+	public func initializeIfNeeded() async throws -> (ServerCapabilities, ServerInfo?) {
```

Comment:
This is a breaking change. And that's totally fine! But, since we're going to change the API, let's change it to something that we can support long-term, by returning an`InitializationResponse`.

Reply:
(just to be really clear, this and returning `initResponse` should be the only modifications required)